index.html

```diff
@@ -13,23 +13,76 @@
     <title>Carrigan's Community</title>
   </head>
   <body>
-    <ul>
+    <ul id="nav">
       <li><a class="active" href="index.html">Home</a></li>
       <li><a href="#gaming">Gaming</a></li>
-      <li><a href="#discussions">Discussions</a></li>
-      <li><a href="./views/calendar.html">Goals & Events</a></li>
-      <li><a href="#community">Community</a></li>
+      <li><a href="#goals">Goals</a></li>
+      <li><a href="#discussion">Discussions</a></li>
+      <li><a href="./views/calendar.html">Events</a></li>
+      <li><a href="#opening">Community</a></li>
       <li><a href="#becca">Becca</a></li>
-      <li><a href="#about">About NhCarrigan</a></li>
+      <li><a href="#whoisNH">About NhCarrigan</a></li>
     </ul>
-    
-    <h1>Welcome to Carrigan's Community!</h1>
-    <p>
-      This page is currently a placeholder. Content will be built and added by
-      our community members, so come back soon to see what they've accomplished!
-    </p>
-    <p>
+    <div id="welcome">
+      <h1>Welcome to Carrigan's Community! 🔥</h1>
+    </div>
+    <main>
+    <section id='opening'>
+      <h2>🔥Our community🔥</h2>
+      <h3>
+        We’re a group of members committed to providing a safe, and comfortable space for anyone to try their hand at coding, and have some fun along the way. As our founding member <b><a href="#whoisNH">NhCarrigan</a></b> is committed to helping members of the <b>freecodecamp</b> community, as well as <b>EddieHub</b> by providing further support to any who need it. We have a staff of moderators who share this vision, come meet them here (link) and check out some of their amazing projects!
```

Comment:
```suggestion
        We’re a group of members committed to providing a safe, and comfortable space for anyone to try their hand at coding, and have some fun along the way. As our founding member <b><a href="#whoisNH">NhCarrigan</a></b> is committed to helping members of the <a href="https://freecodecamp.org"><b>freeCodeCamp</b></a> community, as well as <a href="https://eddiehub.org"><b>EddieHub</b></a> by providing further support to any who need it. We have a staff of moderators who share this vision, come meet them here (link) and check out some of their amazing projects!
```
Add links to FCC and EddieHub?

Reply:
Should I put their website links?

Reply:
Yep!

Reply:
ohk 👍